### ISSUE TITLE
Issue on page /chapters/interp/substitution.html

### ISSUE THREAD

**Issue by Coffee-fan** [cs3110/textbook#96]
I love this entire chapter. The only thing is that as a NOOB I find it hard to follow because for example having a subst stub, creates warnings, which in turn prevents `dune` from completing the build. I found somewhere that you can disable warnings and that does the trick to be able to follow with you the videos and being able to open utop via `make utop`.

 A possible fix would be to add to the `Makefile` some note that in the build target, the call to `dune build` should be changed temporarily to `dune build --profile release`, which allows the build to succeed with warnings, and enables step by step follow along.

**Comment by Coffee-fan** [comment]
BTW, I went back to the beginning of the chapter and put together a small document, starting from `dune init project`, that allowed me to follow you step by step. I did put at some point the ugly `--profile release` which allowed me to follow even when the `subst` stub was generating warnings, thus failing the build. I removed the build hack as soon as the `subst` function was completed.

**Comment by clarksmr** [comment]
Hm, I'm not really following what the issue is here. Possibly I've forgotten something, but, I don't think the textbook ships a file with a "stub" version of `subst`? So I'm not understanding why any disabling of dune warnings is needed for all readers. Perhaps there was something in your own solution that needed it?

**Comment by clarksmr** [comment]
Ah nvm I see in #93 what you mean. Closing this issue as a duplicate.
